src/core/include/openvino/op/softsign.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+
+namespace ov {
+namespace op {
+namespace v9 {
+class OPENVINO_API SoftSign : public Op {
+public:
+    OPENVINO_OP("SoftSign", "opset9");
+    BWDCMP_RTTI_DECLARATION;
+
+    SoftSign() = default;
+    /// \brief Constructs a SoftSign operation.
+    ///
+    /// \param arg Node that produces the input tensor.<br>
+    /// `[d0, ...]`
+    /// Output `[d0, ...]`
+    ///
+    SoftSign(const Output<Node>& arg);
+
+    bool visit_attributes(AttributeVisitor& visitor) override;
+    void validate_and_infer_types() override;
+
+    std::shared_ptr<Node> clone_with_new_inputs(const OutputVector& new_args) const override;
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
Why do we need evaluate for this operation?

Also please use evaluate with ov::Tensor.